<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadContourFile</class>
 <widget class="qMRMLWidget" name="LoadContourFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>311</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="loadContourFileCollapsibleButton">
     <property name="text">
      <string>Load contour file</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="filePathLineEdit">
        <property name="filters">
         <set>ctkPathLineEdit::Executable|ctkPathLineEdit::Files|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
        </property>
        <property name="nameFilters">
         <stringlist comment="s">
          <string>*.ctr</string>
         </stringlist>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputFile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="convertToMMCheckBox">
        <property name="text">
         <string>Convert to mm</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>convertToMM</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="LPSToRASCheckBox">
        <property name="text">
         <string>LPS to RAS</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>LPSToRAS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loadButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
